### ISSUE TITLE
Add basic SANS workflow

### ISSUE THREAD

**Issue by SimonHeybrock** [scipp/beamlime#162]
After #159, replace the placeholder with an actual workflow.

- Keep it minimal for now, compute `IofQ[SampleRun]` without background run subtraction.
- Do not use other inputs files, if it can be avoided. For example mask files and direct-beam files are not needed. This will be dealt with in follow-up.
- Set reasonable defaults for all input parameters. Workflow configuration has not been implemented yet and is not part of this task.
- The workflow should use the `JSONGroup` [that is passed to the workflow](https://scipp.github.io/beamlime/user-guide/getting-started.html#Adding-workflows) as the input sample "file".

**Comment by YooSunYoung** [comment]
I'll clear the milestone from this issue since we implemented #186 instead for the [current milestone](https://github.com/scipp/beamlime/milestone/14)

**Comment by SimonHeybrock** [comment]
Picking this up now, in a modified form. Plan is as follows:

- We have already successfully split event-load from the rest of the load.
- Run workflow to compute all static intermediate results.
- In a loop, load chunk of events and run the workflow until right before normalization. Here we need to accumulate, i.e., add to previous values.
- Compute I(Q) based on accumulated values.

This seems to be possible with some minor and very sensible changes to the existing SANS workflow.
